diff --git a/node_modules/tailwindcss/lib/cli.js b/node_modules/tailwindcss/lib/cli.js
old mode 100755
new mode 100644
diff --git a/node_modules/tailwindcss/lib/lib/setupTrackingContext.js b/node_modules/tailwindcss/lib/lib/setupTrackingContext.js
index 0fd818d..0103b8d 100644
--- a/node_modules/tailwindcss/lib/lib/setupTrackingContext.js
+++ b/node_modules/tailwindcss/lib/lib/setupTrackingContext.js
@@ -29,7 +29,8 @@ function _interop_require_default(obj) {
         default: obj
     };
 }
-let configPathCache = new _quicklru.default({
+const QuickLRU = _quicklru.default || _quicklru;
+let configPathCache = new QuickLRU({
     maxSize: 100
 });
 let candidateFilesCache = new WeakMap();
